doc/classes/LightStorage.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="LightStorage" inherits="RendererLightStorage" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Light storage object.
+		[b]Note:[/b] This is an internal rendering server object, do not instantiate this from script.
+	</brief_description>
+	<description>
+		[b]Note:[/b] This is an internal rendering server object only exposed for GDExtension plugins.
```

Comment:
We should also add a remark here that instances of this class (and any of its subclasses), should only ever be interacted with from the rendering thread.